docs/Antora/modules/getting_started/pages/quick_start_setting_up_cpp.adoc

```diff
@@ -7,19 +7,19 @@ To start working with openDAQ(TM), the requisite binaries are required. They can
 Windows::
 +
 --
-To install openDAQ(TM), download the `opendaq-opendaq_version-win64.exe` installer, run it, and follow its instructions. Make sure to add openDAQ(TM) to the system path to be able to run openDAQ(TM) application examples.
+To install openDAQ(TM), download the `opendaq-latest_dev_opendaq_version-win64.exe` installer, run it, and follow its instructions. Make sure to add openDAQ(TM) to the system path to be able to run openDAQ(TM) application examples.
```

Comment:
just a note: maybe it would be better to use `{opendaq_version}` or something like that so devs would see that its template argument?

Reply:
Yes, but there are still at least other two problems with my previous PR (https://github.com/openDAQ/openDAQ/pull/550):
* In examples we use `set(OPENDAQ_EXAMPLES_VERSION ${OPENDAQ_PACKAGE_VERSION})` (https://github.com/openDAQ/openDAQ/blob/main/examples/CMakeLists.txt) which means you can't build the examples (which are also available as a standalone download on https://docs-dev.opendaq.com/) without building the whole project
* https://opendaq.github.io/ will show the placeholders either way

So I'm thinking we'll have to revert these changes... Because if you look at the `antora-playbook.yml` for that docs page I see no way to circumvent hard-coded strings (https://github.com/openDAQ/opendaq.github.io/blob/master/antora-playbook.yml)

Reply:
i see in the official site that ascii doc supports Attributes. maybe it will help you [Attributes and substitutions](https://docs.asciidoctor.org/asciidoc/latest/syntax-quick-reference/#attributes-and-substitutions)

Reply:
This is true, but `config.adoc` would still have to be hard-coded, correct?

Reply:
well, docs/version_injector.py can then modify only this file :)

Reply:
True, I will change it like you suggested. But this wont work for: https://opendaq.github.io/